ucm2/MediaTek/mt8390-evk/HiFi.conf

```diff
@@ -0,0 +1,217 @@
+SectionVerb {
+	EnableSequence [
+		cset "name='HDMI_OUT_MUX' Connect"
+		cset "name='DPTX_OUT_MUX' Disconnect"
+		cset "name='O000 I000 Switch' on"
+		cset "name='O001 I001 Switch' on"
+		cset "name='O002 I004 Switch' on"
+		cset "name='O003 I006 Switch' on"
+		cset "name='O034 I168 Switch' on"
+		cset "name='O035 I169 Switch' on"
+		cset "name='O036 I012 Switch' on"
+		cset "name='O037 I013 Switch' on"
+		cset "name='O040 I002 Switch' on"
+		cset "name='O041 I003 Switch' on"
+		cset "name='O048 I022 Switch' on"
+		cset "name='O049 I023 Switch' on"
+		cset "name='O050 I024 Switch' on"
+		cset "name='O051 I025 Switch' on"
+		cset "name='O052 I026 Switch' on"
+		cset "name='O053 I027 Switch' on"
+		cset "name='O054 I028 Switch' on"
+		cset "name='O055 I029 Switch' on"
+		cset "name='O176 I070 Switch' on"
+		cset "name='O177 I071 Switch' on"
+	]
+
+	DisableSequence [
+		cset "name='HDMI_OUT_MUX' Disconnect"
+		cset "name='DPTX_OUT_MUX' Disconnect"
+		cset "name='O000 I000 Switch' off"
+		cset "name='O001 I001 Switch' off"
+		cset "name='O002 I004 Switch' off"
+		cset "name='O003 I006 Switch' off"
+		cset "name='O034 I168 Switch' off"
+		cset "name='O035 I169 Switch' off"
+		cset "name='O036 I012 Switch' off"
+		cset "name='O037 I013 Switch' off"
+		cset "name='O040 I002 Switch' off"
+		cset "name='O041 I003 Switch' off"
+		cset "name='O048 I022 Switch' off"
+		cset "name='O049 I023 Switch' off"
+		cset "name='O050 I024 Switch' off"
+		cset "name='O051 I025 Switch' off"
+		cset "name='O052 I026 Switch' off"
+		cset "name='O053 I027 Switch' off"
+		cset "name='O054 I028 Switch' off"
+		cset "name='O055 I029 Switch' off"
+		cset "name='O176 I070 Switch' off"
+		cset "name='O177 I071 Switch' off"
+	]
+}
+
+SectionDevice."Line4" {
+	Comment "I2S output"
+
+	Value {
+		PlaybackPriority 100
+		PlaybackChannels 8
+		PlaybackPCM "hw:${CardId},6"
+	}
+}
+
+SectionDevice."Line2" {
```

Comment:
Sort devices, Line1, Line2, Line3 - there should not be gap. Also, usually the first device is the one with the highest priority.

Reply:
Got it.
If there are PCM input/output devices and I2S input/output devices, is it necessary to sort all the devices of the same kind together? Or just make sure that the number suffixes are continuous?

Reply:
Additionally, I would like to ask about the purpose of the Priority values.
Will it automatically switch to the device with the highest priority when the device in use becomes unavailable?
According to my test results, it seems that the priority value is not related to the default device when the system starts up.

Reply:
The sound server should use priorities to select the device, but the question is, if PA code does the right job. Also, when Jack is not defined, the automatic device selection behaves differently.